lib/geojsonLayerController.js

```diff
@@ -62,6 +65,7 @@ function buildConfigProcessingOptions(layer, config) {
     iconStylePresenter: config.stylePresenterFactory(nn(layer)('styles.icon').val),
```

Comment:
You mean this module that is used just a few lines below? :stuck_out_tongue_winking_eye: